File: 11. Pendalaman String/B. Pembuangan String/b_pembuangan_string.go
package main

import (
  "fmt"
  "strings"
)

func main() {
  var S, T string
  fmt.Scan(&S, &T)
  for strings.Contains(S, T) {
    S = strings.ReplaceAll(S, T, "")
  }
  fmt.Println(S)
}

File: 11. Pendalaman String/B. Pembuangan String/b_pembuangan_string.java
import java.util.Scanner;

class b_pembuangan_string {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        String line = scan.nextLine();
        scan.close();
        String[] S = line.split(" ");
        while (S[0].contains(S[1])) {
            S[0] = S[0].replaceAll(S[1], "");
        }
        System.out.println(S[0]);
    }
}